### PR TITLE
w01d01

### PR DESCRIPTION
**Pull request by BreanaMarie**
comfort:3
completeness:5

### CODE REVIEW

**Comment by BreanaMarie** [comment]
Additional changes to W01D01 homework.
comfort level:4
completness:5

File: BreanaMarie/passion_project.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title></title>
+</head>
+<body>
+<p>
+	I have several project i want to look into, however the largest and most exciting one is an interactive website that requires a login for users that prevents unregistered users from accessing all content, and a specialized set of forms.  The forms are my biggest focus, each form needs to be dynamic, and based off of the type of input, a numeric value is given that will interact with running total specific to the user.  This would be similar to how a shopping cart keeps track of how much money is spent.  </br>
+	Each time a user logs in, they can access their total "points" and can "purchase" perks with those points from a list on the form.  The form would need to changed or update every time a field is manipulated.  However, once all the points are used, the form needs to be inactive.   I also want to be able to add more points to a users total coffer, but prevent any fields from being modified once the points have been used to purchase something on the form.  </br>
```
Comment (tgaff) [inline BreanaMarie/passion_project.html]: wow, good detail!

File: BreanaMarie/day1.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+		</li>
+		<li>
+			Question 2: not so much a question but after reading the CSS information and the comparison to a picture frame i started thinking about liquid sites and how setting specific sizes of blocks would impact the various size screens.  Is is better to design in liquid format or in stactic format?
+		</li>
+		<li>
+			Question 3: not much else on my mind related to the HTML and CSS articles.
+		</li>
+		<li> Additional comment: kudos to the writer Cecily Carver.  The idea that "the difference between new coders and experienced coders is faith ..."  Although my time coding is relatively short, I know how important it was to find that first break in my code and fix it was.  That bit of confidence that is  needed to know that you can find that little missing ";" or misspelled variable and correct it to make it all better ... or not and you have to find the next break. 
+		</li>
+		<li> Additional comment: but after watching the video about the power of belief I found the message positive, and overall interesting that a small change in sentence of “I can’t” to “I can’t yet” can impact ones ability to learn and be interested in the material  they are exposed to.</li>
```
Comment (tgaff) [inline BreanaMarie/day1.html]: nice :camel:

**Comment by tgaff** [comment]
well done :-)

**Pull request merged**